### ISSUE TITLE
Showing incorrrect num lock status

### ISSUE THREAD

**Issue by RishiCodes** [ModernFlyouts-Community/ModernFlyouts#690]
Whenever I unlock Number Pad, flyout shows "Num Lock is On" when it's  actually off and same issue when I lock number pad. It's showing opposite staus.

**Comment by JohnLGalt** [comment]
Probably going to need some information - like OS, computer model, OEM supplied apps running in the background, etc.

**Comment by rishi-shekhar** [comment]
Edition Windows 10 Home Single Language
Version 21H1
OS build 19043.1083
Experience Windows Feature Experience Pack 120.2212.3530.0

Processor Intel(R) Core(TM) i5-7200U CPU @ 2.50GHz   2.70 GHz
Installed RAM 4.00 GB
System type 64-bit operating system, x64-based processor
Pen and touch No pen or touch input is available for the display

That's all I know. I don't know about OEM

Thank You.

**Comment by Cyberdroid1** [comment]
@rishi-shekhar Can you verify if the issue #91 is similar to what you face?

**Comment by rishi-shekhar** [comment]
No, It’s not.  From: Prakhar BSent: 12 July 2021 11:54To: ModernFlyouts-Community/ModernFlyoutsCc: Rishi Shekhar; MentionSubject: Re: [ModernFlyouts-Community/ModernFlyouts] Showing incorrrect num lock status ***@***.*** Can you verify if the issue #91 is similar to what you face?—You are receiving this because you were mentioned.Reply to this email directly, view it on GitHub, or unsubscribe.
